build.assets/tooling/cmd/please-run.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env sh
+
+# This script is a helper that tells developers what generated content is out of date
+# and which command to run.
+# When running on GitHub actions, the script will also create an error in the PR and
+# collapse the diff to improve readability.
+
+set -eu
+
+KIND="$1"
```

Comment:
Check that you've got the correct number of parameters before addressing them (this will fail because of set -u).

```shell
if [ #$ -ne 2 ]; then
  echo "Usage: $0 <kind> <generate command>" >&2
  exit 1
fi
```